app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::API
+  rescue_from(ActionController::BadRequest, with: :render_bad_request_response)
+
+  def render_bad_request_response(error)
+    render(json: { errors: [{ detail: "Bad request: #{error.message}" }] }, status: :bad_request)
```

Comment:
The JSON object is returned when an invalid `datetime` query param is sent through the request